File: android/versioned-abis/expoview-abi40_0_0/src/main/java/abi40_0_0/expo/modules/brightness/BrightnessPackage.java
package abi40_0_0.expo.modules.brightness;

import android.content.Context;

import java.util.Collections;
import java.util.List;

import abi40_0_0.org.unimodules.core.BasePackage;
import abi40_0_0.org.unimodules.core.ExportedModule;

public class BrightnessPackage extends BasePackage {
  @Override
  public List<ExportedModule> createExportedModules(Context context) {
    return Collections.singletonList((ExportedModule) new BrightnessModule(context));
  }
}

File: android/versioned-abis/expoview-abi43_0_0/src/main/java/abi43_0_0/expo/modules/brightness/BrightnessPackage.kt
package abi43_0_0.expo.modules.brightness

import android.content.Context
import abi43_0_0.expo.modules.core.BasePackage
import abi43_0_0.expo.modules.core.ExportedModule

class BrightnessPackage : BasePackage() {
  override fun createExportedModules(context: Context): List<ExportedModule> {
    return listOf(BrightnessModule(context))
  }
}
